api/server/router/system/filter_test.go

```diff
@@ -0,0 +1,62 @@
+package system
+
+import (
+	"testing"
+
+	"github.com/docker/docker/api/types/filters"
+)
+
+func TestFilterEvents(t *testing.T) {
```

Comment:
This test is not actually testing anything about `getEvents()`. It's testing the filters type.

If that type doesn't already have it's own tests maybe we could move these tests to that package?

I think we still need a test for `getEvents()`